### ISSUE TITLE
Incompleteable quest

### ISSUE THREAD

**Issue by GoogleCodeExporter** [alistairreilly/andors-trail#112]
```
What steps will reproduce the problem?
1.Accept quest to kills Unzel
2.Listen to Unzels story accept to kill Vacor
3.Kill Unzel or Vacor
4.Return to complete quest, select background and now kill the other.
5. Quest can not be completed now

What is the expected output? What do you see instead?

Once you kill either Unzel or Vacor the quest Giver should just allow you to 
complete the quest.  Instead if you decide to kill both there is no way to 
complete the quest.

What version of the product are you using? On what operating system?

Current version(12/29/10), samsung galaxy


Please provide any additional information below.


```

Original issue reported on code.google.com by `justin.m...@gmail.com` on 30 Dec 2010 at 1:29

**Comment by GoogleCodeExporter** [comment]
```
I realized the same thing. I saved beforehand so I actually could complete the 
quest but the flaw is there. I'm using v6.7 of Andors Trail on an LG Ally v2.1.
```

Original comment by `doug_y...@hotmail.com` on 30 Dec 2010 at 5:19

**Comment by GoogleCodeExporter** [comment]
```
Could also be a feature. ;-)
So the Quest has 3 possible endings instead of 2. :-D
```

Original comment by `SamuelPl...@gmail.com` on 30 Dec 2010 at 7:45

**Comment by GoogleCodeExporter** [comment]
```
Just as long as the quest says it's actually complete... I'd be a fan of 
getting 2 rewards. 
```

Original comment by `Captain8...@gmail.com` on 30 Dec 2010 at 4:56

**Comment by GoogleCodeExporter** [comment]
```
Thanks for the report. Well done, I did not think of that option. :)

However we will not add this as a third option to that particular quest. If you 
want to kill both of them then please go ahead, but your quest will stay 
unresolved since you did not help any of them. Depending on who you choose to 
help, you will see some consequenses later in the game.
```

Original comment by `oskar.wi...@gmail.com` on 9 Jan 2011 at 7:51

**Comment by GoogleCodeExporter** [comment]
```
Just make sure you think about the possibility "helped neither" :)
```

Original comment by `surrano` on 9 Jan 2011 at 2:44

**Comment by GoogleCodeExporter** [comment]
```
This should be fixed for the next release.
```

Original comment by `sdeva...@gmail.com` on 10 Feb 2011 at 1:46
- Changed state: **Accepted**
- Added labels: **Priority-High**, **Milestone-0.6.8**
- Removed labels: **Priority-Medium**

**Comment by GoogleCodeExporter** [comment]
```
Any estimate when the next release will be available?

I'm looking forward to getting the heartstone and going to Nor.

By the way, where is that ladies husband that seems to be slacking off?
```

Original comment by `rubico...@hotmail.com` on 10 Feb 2011 at 2:04

**Comment by GoogleCodeExporter** [comment]
```
[deleted comment]
```

**Comment by GoogleCodeExporter** [comment]
```
Check out the forums at http://andors.techby2guys.com for the latest 
information on v0.6.8
```

Original comment by `sdeva...@gmail.com` on 10 Feb 2011 at 10:20

**Comment by GoogleCodeExporter** [comment]
```
Released in v0.6.8 . Closing.
```

Original comment by `oskar.wi...@gmail.com` on 24 Feb 2011 at 9:35
- Changed state: **Fixed**
